manifests/init.pp

```diff
@@ -152,7 +152,8 @@
   $refresh              = $uchiwa::params::refresh,
   $sensu_api_endpoints  = $uchiwa::params::sensu_api_endpoints,
   $users                = $uchiwa::params::users,
-  $auth                 = $uchiwa::params::auth
+  $auth                 = $uchiwa::params::auth,
+  $ssl                  = $uchiwa::params::ssl,
```

Comment:
Can you add a docstring to explain how to use this? Bonus points for a readme update